js/app.js

```diff
@@ -0,0 +1,135 @@
+window.onload = function () {
+  //add listener to request button
+  let reqBtn = document.getElementById('requestResourceButton');
+  reqBtn.addEventListener('click', runRequest);
+
+  function runRequest() {
+    //gets data from user inputs
+    let input = document.getElementById('resourceId').value;
+    let type = document.getElementById('resourceType').value;
+
+    //joins input to make url
+    let url = 'https://swapi.co/api/' + type + '/' + input + '/';
+
+    let oReq = new XMLHttpRequest();
+    oReq.addEventListener('load', reqListen);
+    oReq.open('GET', url);
+    oReq.send();
+
+    //listener for original request
+    function reqListen() {
+      let contain = document.getElementById('contentContainer');
+      contain.innerHTML = '';
+
+      //error handler
+      if (this.status !== 200) {
+        let errorH2 = document.createElement('h2');
+        contain.appendChild(errorH2);
+        errorH2.innerHTML = this.responseURL + this.responseText;
+        return;
+      }
+
+      //parses response text into new variable
+      let response = JSON.parse(this.responseText);
+
+      //if-else statement seperating operations depending on user input
+      if (type === 'people') {
+
```

Comment:
Typically there are no newlines between the if statement and the next line of code.